### PR TITLE
refactor(runtime-fuzzer): Change required size for gear calls data

### PR DESCRIPTION
**Pull request by SabaunT**
Also this PR changes sizes of random input seed for testing fuzzer and max corpus size.

### CODE REVIEW

File: utils/runtime-fuzzer/src/gear_calls.rs [hunk 1]
```diff
@@ -51,6 +52,10 @@ static_assertions::const_assert!(MAX_PAYLOAD_SIZE <= gear_core::message::MAX_PAY
 const MAX_SALT_SIZE: usize = 512;
 static_assertions::const_assert!(MAX_SALT_SIZE <= gear_core::message::MAX_PAYLOAD_SIZE);
 
+const ID_SIZE: usize = mem::size_of::<ProgramId>();
+const GAS_AND_VALUE_SIZE: usize = mem::size_of::<(u64, u128)>();
+const AUXILIARY_SIZE: usize = 512;
```
Comment (mertwole) [inline utils/runtime-fuzzer/src/gear_calls.rs:57]:
```suggestion
// Used to make sure that generators will not exceed `Unstructured` size as it's used not only
// to generate things like wasm code or message payload but also to generate some auxiliary
// data, for example index in some vec.
const AUXILIARY_SIZE: usize = 512;
```

File: utils/runtime-fuzzer/src/gear_calls.rs [hunk 2]
```diff
@@ -51,6 +52,10 @@ static_assertions::const_assert!(MAX_PAYLOAD_SIZE <= gear_core::message::MAX_PAY
 const MAX_SALT_SIZE: usize = 512;
 static_assertions::const_assert!(MAX_SALT_SIZE <= gear_core::message::MAX_PAYLOAD_SIZE);
 
+const ID_SIZE: usize = mem::size_of::<ProgramId>();
+const GAS_AND_VALUE_SIZE: usize = mem::size_of::<(u64, u128)>();
```
Comment (mertwole) [inline utils/runtime-fuzzer/src/gear_calls.rs:56]: I think it can be covered by the `AUXILIARY_SIZE` bytes

**Review by mertwole — APPROVED** [review]

File: docker/runtime-fuzzer/scripts/fuzzer.sh [hunk 3]
```diff
@@ -70,7 +70,7 @@ function start_container_post {
 		rustup component add llvm-tools-preview && \
 		rustup component add --toolchain nightly llvm-tools-preview && \
 		cargo fuzz coverage --release --sanitizer=none main /corpus/main -- \
-        -rss_limit_mb=8192 -max_len=35000000 -len_control=0 && \
+        -rss_limit_mb=8192 -max_len=20000000 -len_control=0 && \
```
Comment (breathx) [inline docker/runtime-fuzzer/scripts/fuzzer.sh:73]: what about introducing .sh file with consts like this?

**Review by breathx — APPROVED** [review]

File: utils/runtime-fuzzer/src/gear_calls.rs (hunk 1 above)
Comment (SabaunT) [reply utils/runtime-fuzzer/src/gear_calls.rs:60]: done

File: utils/runtime-fuzzer/src/gear_calls.rs (hunk 2 above)
Comment (SabaunT) [reply utils/runtime-fuzzer/src/gear_calls.rs:56]: I'd leave bytes for id and gas for more clarity

File: docker/runtime-fuzzer/scripts/fuzzer.sh (hunk 3 above)
Comment (SabaunT) [reply docker/runtime-fuzzer/scripts/fuzzer.sh:73]: partially done,
for docker related scripts - https://github.com/gear-tech/gear/issues/3397

**Pull request merged**